x-pack/plugins/security_solution/public/flyout/document_details/right/components/highlighted_fields_cell.test.tsx

```diff
@@ -95,23 +107,32 @@ describe('<HighlightedFieldsCell />', () => {
     expect(getByTestId(HIGHLIGHTED_FIELDS_AGENT_STATUS_CELL_TEST_ID)).toBeInTheDocument();
   });
 
-  it('should render sentinelone agent status cell if field is agent.status and origialField is observer.serial_number', () => {
-    (useGetSentinelOneAgentStatus as jest.Mock).mockReturnValue({
-      isFetched: true,
-      isLoading: false,
-    });
-    const { getByTestId } = render(
-      <TestProviders>
-        <HighlightedFieldsCell
-          values={['value']}
-          field={'agent.status'}
-          originalField="observer.serial_number"
-        />
-      </TestProviders>
-    );
-
-    expect(getByTestId(HIGHLIGHTED_FIELDS_AGENT_STATUS_CELL_TEST_ID)).toBeInTheDocument();
-  });
+  // TODO:
+  // simplify when agentStatusClient FF is enabled/removed
```

Comment:
same here, any reason to write the TODO on 2 lines instead of 1?

Reply:
97f6092864ff8adda824dc41ee2d28f3a23cd539